webapp/utils/constants.jsx

```diff
@@ -849,7 +849,8 @@ export const Constants = {
     MENTION_MEMBERS: 'mention.members',
     MENTION_NONMEMBERS: 'mention.nonmembers',
     MENTION_SPECIAL: 'mention.special',
-    DEFAULT_NOTIFICATION_DURATION: 5000
+    DEFAULT_NOTIFICATION_DURATION: 5000,
+    STATUS_INTERVAL: 15000
```

Comment:
back of hand math.  20K users / 15 is 1333 requests a second.  I think it will cause issues :)